lam4doc/entrypoints/api/handlers.py

```diff
@@ -39,3 +44,63 @@ def generate_lam_report() -> tuple:
     except Exception as e:
         logger.exception(str(e))
         raise InternalServerError(str(e))  # 500
+
+
+def upload_rdfs(body: dict, lam_properties_document: FileStorage = None, lam_classes_document: FileStorage = None,
+                celex_classes_document: FileStorage = None) -> tuple:
+    """
+    API method for uploading 3 RDF files.
+    :return: nothing
+    """
+    logger.debug('Entering upload_rdfs')
+
+    dataset_name = body.get("dataset_name")
```

Comment:
isn't this the responsibility of the connexion, to check the mandatory parameters? 
I would delegate this functionality.

Reply:
As far as I understood connexion is quite problematic and we might change it. I'd like to keep the code so that the parameter checks remain in place for anyone who would call this method.